packages/transformer/src/IModelExporter.ts

```diff
@@ -315,6 +320,7 @@ export class IModelExporter {
         ? accessTokenOrOpts.changedInstanceIds
         : undefined)
       ?? await ChangedInstanceIds.initialize({ ...opts, iModel: this.sourceDb });
+    this.handleEntityRecreations();
```

Comment:
we need to think about how to do this in the transformer, not here. The exporter doesn't know/care about element "recreations" because it has no relationship to a target iModel, the transformer superimposes that relationship